.node-version

```diff
@@ -1 +1 @@
-16.15.1
+18.19.0
```

Comment:
Some packages, such as the latest version of Stylelint, require Node version >= 18. This version is an LTS version and works seamlessly with the rest of the project.